### ISSUE TITLE
readthedocs documentation often fail to build

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#135]
More than a half of builds on eli5.readthedocs.io fail for some reason. Some builds are passing, using the same configuration.

**Comment by kmike** [comment]
No successful builds recently.. It looks like a release blocker, so I'm adding 0.3 milestone.

**Comment by lopuhin** [comment]
It seems travis is not happy now: https://travis-ci.org/TeamHG-Memex/eli5/builds/190885579

**Comment by kmike** [comment]
Ouch, let mefix that.

**Comment by kmike** [comment]
Fixed in https://github.com/TeamHG-Memex/eli5/commit/fff8352650013824d1c9a1ec0e517b054ff126d2

**Comment by lopuhin** [comment]
Yay, thanks!
